Software/Source/BrstPktCnt.py

```diff
@@ -0,0 +1,50 @@
+import json
+#Global Variable
+a=0
+b=0
+c=0
+d=0
+e=0

```

Comment:
Use meaningful variable names.
Using meaningful variable names is instructed in previous comments also. The comments are applicable not only to the current code, but also for your future codes. Please remember the feedbacks, comments and recommendations and incorporate them in your learnings.